cli/cmd/check.go

```diff
@@ -268,20 +266,25 @@ func getExtensionCheckFlags(lf *pflag.FlagSet) []string {
 }
 
 func renderInstallManifest(ctx context.Context) (*charts.Values, string, error) {
+	// Create the default values.
```

Comment:
the placement of this comment is a bit misleading since the API isn't used in the creation of the default values.